proto/archway/rewards/v1beta1/query.proto

```diff
@@ -98,3 +103,32 @@ message QueryEstimateTxFeesResponse {
     (gogoproto.nullable) = false
   ];
 }
+
+// BlockTracking is the tracking information for a block.
+message BlockTracking {
+  option (gogoproto.goproto_stringer) = false;
+
+  // inflation_rewards defines the inflation rewards for the block.
+  BlockRewards inflation_rewards = 1 [
+    (gogoproto.nullable) = false
+  ];
+
+  // tx_rewards defines the transaction rewards for the block.
+  repeated TxRewards tx_rewards = 2 [
+    (gogoproto.nullable) = false
+  ];
+}
+
+// QueryCurrentRewardsRequest is the request for Query.CurrentRewards.
+message QueryCurrentRewardsRequest {
```

Comment:
Can we add pagination here? https://github.com/cosmos/cosmos-sdk/blob/main/types/query/pagination.go#L52

Reply:
It returns the total amount of rewards (a single `sdk.Coins` field). Do we need a pagination here?

Reply:
Got it (didn't read the comment below).